js/compressed-token/src/program.ts

```diff
@@ -130,7 +344,10 @@ export class CompressedTokenProgram {
     /** @internal */
     static get deriveCpiAuthorityPda(): PublicKey {
         const [address, _] = PublicKey.findProgramAddressSync(
-            [Buffer.from('cpi_authority')],
+            [
+                Buffer.from('cpi_authority'),
```

Comment:
what about creating a constant for this or even taking the constant from the idl?

Reply:
made constant